### PR TITLE
Spanish translations

### PR DESCRIPTION
**Pull request by Moonstroke82**
When merged this pull request will:
Add Spanish translation for
Dogtags
UI
AI
Vehicles
Weapons
Zeus

Turkish translation for
UI (incomplete)
AI (incomplete)

### CODE REVIEW

**Comment by commy2** [comment]
How come you speak Turkish and Spanish?

**Comment by Moonstroke82** [comment]
Haha my buddy and best man is Turkish and my wife used to life in Spain :)

File: addons/ai/stringtable.xml [hunk 1]
```diff
@@ -13,6 +13,8 @@
             <Polish>Podano błędną pozycję.</Polish>
             <Russian>Указана неверная позиция</Russian>
             <Portuguese>Posição inválida fornecida.</Portuguese>
+            <Spanish>Posición no válida proporcionada</Spanish>
```
Comment (TheMagnetar) [inline addons/ai/stringtable.xml]:
```suggestion
            <Spanish>Posición inválida proporcionada</Spanish>
```

File: addons/common/stringtable.xml [hunk 2]
```diff
@@ -529,6 +531,7 @@
             <Polish>Sprawdź wszystkie PBO</Polish>
             <Russian>Проверять все PBO</Russian>
             <Portuguese>Checar Todos os PBOs</Portuguese>
+            <Spanish>Comprobar PBO todo</Spanish>
```
Comment (TheMagnetar) [inline addons/common/stringtable.xml]:
```suggestion
            <Spanish>Comprobar todos los PBO</Spanish>
```

File: addons/dogtags/stringtable.xml [hunk 3]
```diff
@@ -84,6 +89,7 @@
             <Chinese>已經有人把他的兵籍牌拿走了...</Chinese>
             <Chinesesimp>已经有人把他的兵籍牌拿走了...</Chinesesimp>
             <Portuguese>Alguém já pegou essa dogtag...</Portuguese>
+            <Spanish>Alguien más ha tomado la placa de identificación</Spanish>
```
Comment (TheMagnetar) [inline addons/dogtags/stringtable.xml]:
```suggestion
            <Spanish>Alguien más ha tomado la placa de identidad</Spanish>
```

File: addons/ui/stringtable.xml [hunk 4]
```diff
@@ -69,6 +72,7 @@
             <Italian>Permette al client di modificare la propria UI.</Italian>
             <Chinesesimp>允许客户端自行调整使用者介面</Chinesesimp>
             <Chinese>允許客戶端自行調整使用者介面</Chinese>
+            <Spanish>Permitir al cliente modificar su IU.</Spanish>
```
Comment (TheMagnetar) [inline addons/ui/stringtable.xml]:
```suggestion
            <Spanish>Permitir al cliente modificar su interfaz.</Spanish>
```

File: addons/ui/stringtable.xml [hunk 5]
```diff
@@ -209,6 +222,7 @@
             <Italian>Contatore Munizioni</Italian>
             <Chinesesimp>弹药数量</Chinesesimp>
             <Chinese>彈藥數量</Chinese>
+            <Spanish>Cuenta de munición</Spanish>
```
Comment (TheMagnetar) [inline addons/ui/stringtable.xml]:
```suggestion
            <Spanish>Cantidad de munición</Spanish>
```

File: addons/ui/stringtable.xml [hunk 6]
```diff
@@ -223,6 +237,7 @@
             <Italian>Contatore Caricatore</Italian>
             <Chinesesimp>弹匣数量</Chinesesimp>
             <Chinese>彈匣數量</Chinese>
+            <Spanish>Recuento de revistas</Spanish>
```
Comment (TheMagnetar) [inline addons/ui/stringtable.xml]:
```suggestion
            <Spanish>Recuento de cargadores</Spanish>
```

File: addons/ui/stringtable.xml [hunk 7]
```diff
@@ -251,6 +267,7 @@
             <Italian>Contatore Lanciabili</Italian>
             <Chinesesimp>投掷物数量</Chinesesimp>
             <Chinese>投擲物數量</Chinese>
+            <Spanish>Cuenta arrojable</Spanish>
```
Comment (TheMagnetar) [inline addons/ui/stringtable.xml]:
```suggestion
            <Spanish>Cantidad arrojable</Spanish>
```

File: addons/vehicles/stringtable.xml [hunk 8]
```diff
@@ -75,20 +77,23 @@
...
             <Portuguese>Desativar desligamento automático do motor</Portuguese>
+            <Spanish>Desactivar apagado automático del motor</Spanish>
         </Key>
         <Key ID="STR_ACE_Vehicles_SettingKeepEngineRunningDesc">
             <English>Prevent the automatic shut-off of the engine when exiting vehicles.</English>
             <German>Verhindere das automatische Abschalten des Motors beim Verlassen des Fahrzeugs.</German>
             <Japanese>車両から降りた時のエンジンの自動停止を防ぎます。</Japanese>
             <Russian>Запрещает автоматическое выключение двигателя при выходе из транспорта</Russian>
             <Portuguese>Previne que o motor do veículo seja desligado automaticamente quando o motorista sair.</Portuguese>
+            <Spanish>Evite el apagado automático del motor al salir de los vehículos.</Spanish>
```
Comment (TheMagnetar) [inline addons/vehicles/stringtable.xml]:
```suggestion
            <Spanish>Desactivar el apagado automático del motor al salir de los vehículos.</Spanish>
```

File: addons/zeus/stringtable.xml [hunk 9]
```diff
@@ -700,6 +702,7 @@
             <Chinesesimp>传送全部小队成员</Chinesesimp>
             <Chinese>傳送全部小隊成員</Chinese>
             <Portuguese>Teleporta todas as unidades do grupo</Portuguese>
+            <Spanish>Teletransporta todas las unidades en grupo</Spanish>
```
Comment (TheMagnetar) [inline addons/zeus/stringtable.xml]:
```suggestion
            <Spanish>Teletransporta todas las unidades en el grupo</Spanish>
```

File: addons/zeus/stringtable.xml [hunk 10]
```diff
@@ -744,6 +748,7 @@
             <Chinesesimp>搜索附近的建筑物</Chinesesimp>
             <Chinese>搜索附近的建築物</Chinese>
             <Portuguese>Procurar construções próximas</Portuguese>
+            <Spanish>Buscar edificio cercano</Spanish>
```
Comment (TheMagnetar) [inline addons/zeus/stringtable.xml]:
```suggestion
            <Spanish>Buscar edificios cercanos</Spanish>
```

File: addons/zeus/stringtable.xml [hunk 11]
```diff
@@ -802,6 +807,7 @@
             <Chinese>切換模擬</Chinese>
             <Russian>Вкл/Выкл симуляцию</Russian>
             <Portuguese>Alternar Simulação</Portuguese>
+            <Spanish>Simulación de alternancia</Spanish>
```
Comment (TheMagnetar) [inline addons/zeus/stringtable.xml]:
```suggestion
            <Spanish>Activar  simulación</Spanish>
```

File: addons/zeus/stringtable.xml [hunk 12]
```diff
@@ -954,6 +960,7 @@
             <Chinesesimp>单位必须属于一个合适的一边</Chinesesimp>
             <Chinese>單位必須屬於一個合適的一邊</Chinese>
             <Portuguese>Unidade deve pertencer á um lado apropriado</Portuguese>
+            <Spanish>La unidad debe pertenecer a un lado apropiado</Spanish>
```
Comment (TheMagnetar) [inline addons/zeus/stringtable.xml]:
```suggestion
            <Spanish>La unidad debe pertenecer al bando apropiado</Spanish>
```

File: addons/common/stringtable.xml [hunk 13]
```diff
@@ -541,6 +544,7 @@
             <Polish>Sprawdź białą listę PBO</Polish>
             <Russian>Белый список для проверки PBO</Russian>
             <Portuguese>Lista Branca de PBO</Portuguese>
+            <Spanish>Comprobar PBO lista blanca</Spanish>
```
Comment (TheMagnetar) [inline addons/common/stringtable.xml]:
```suggestion
            <Spanish>Comprobar lista blanca de PBOs</Spanish>
```

File: addons/ai/stringtable.xml [hunk 14]
```diff
@@ -52,6 +58,8 @@
             <Polish>Nie znaleziono budynku.</Polish>
             <Russian>Здание не найдено</Russian>
             <Portuguese>Nenhuma construção encontrada.</Portuguese>
+            <Spanish>No se encontró edificio.</Spanish>
```
Comment (TheMagnetar) [inline addons/ai/stringtable.xml]:
```suggestion
            <Spanish>No se ha encontrado ningún edificio.</Spanish>
```

**Pull request merged**